docs/crio.8.md

```diff
@@ -115,6 +116,13 @@ If `hooks_dir` is unset, CRI-O will currently default to `/usr/share/containers/
 
 **--log-size-max**="": Maximum log size in bytes for a container (default: -1 (no limit)). If it is positive, it must be >= 8192 (to match/exceed conmon read buffer).
 
+<<<<<<< HEAD
```

Comment:
Looks like you've merge bits left behind here.

Reply:
good eye!